.github/workflows/ci.yml

```diff
@@ -31,7 +31,9 @@ jobs:
     # See: https://github.com/gradle/actions/blob/main/setup-gradle/README.md
     - name: Setup Gradle
       uses: gradle/actions/setup-gradle@v4.0.0
-
+    # Setup for misc tests
+    - name: Install shellcheck and checkbashisms
```

Comment:
I see this step in most CI jobs, but NOT in the `misc` jobs where it is needed. I don't see why...

Reply:
@wmdietl The installations are put int wrong place. Now it is corrected. Could you take a look when the `ci-lint-diff` is failing? I think I see it from previous PR but can remember which one.